DEP Tasks/Trainers Task/JSCore/Task 5/Array-Helpers.js

```diff
@@ -2,53 +2,30 @@
 // https://www.codewars.com/kata/array-helpers
 
 Array.prototype.square = function () {
-  var arr = [];
-  for (var i = 0; i < this.length; i++) {
-    arr.push(this[i] * this[i]);
-  }
-  return arr;
+  return this.map(function (n) {
+    return n * n;
+  });
 }
-
 Array.prototype.cube = function () {
-  var arr = [];
-  for (var i = 0; i < this.length; i++) {
-    arr[i] = this[i] * this[i] * this[i];
-  }
-  return arr;
+  return this.map(function (n) {
```

Comment:
Use arrow functions instead of anonymous functions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions